server/bundles/io.cloudbeaver.server/schema/service.core.graphqls

```diff
@@ -150,7 +150,7 @@ type ServerConfig {
     supportedLanguages: [ ServerLanguage! ]!
     services: [ WebServiceConfig ]
     productConfiguration: Object!
-    productInfo: ProductInfo!
+    productInfo: ProductInfo
```

Comment:
This is not backward compatible. We can't make this type of change in the schema